test/functorch/test_control_flow.py

```diff
@@ -650,8 +654,8 @@ def f(x, y):
 
         x = torch.randn(4)
         with self.assertRaisesRegex(
-            CondOpArgsMismatchError,
-            "Expected to return same number of outputs but got",
+            torch._dynamo.exc.UserError,
+            "Expected branch out type to be a single tensor",
```

Comment:
`branch out type` isn't very clear. Maybe `Expected branch to return a single tensor`?